radio/src/tasks_arm.cpp

```diff
@@ -145,7 +148,14 @@ void mixerTask(void * pdata)
       if (t0 > maxMixerDuration) maxMixerDuration = t0 ;
     }
 
-    CoTickDelay(1);  // 2ms for now
+    int delay = 10; // 20ms default
+    if (GET_MIXER_DELAY(0) >= 0)
+      delay = min(delay, GET_MIXER_DELAY(0));

```

Comment:
You have two calls to the GET_MIXER_DELAY() which don't get optimized (64 bit substraction), so we end up doing substraction two times. Maybe get the value in temporary variable and work with that value:
```c
    int delay = 10; // 20ms default
    int moduleDelay = GET_MIXER_DELAY(0);
    if (moduleDelay >= 0)
      delay = min(delay, moduleDelay);
#if NUM_MODULES >= 2
    moduleDelay = GET_MIXER_DELAY(1);
    if (moduleDelay >= 0)
      delay = min(delay, moduleDelay);
#endif
```